scripts/jtsync/test_jtsync.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 TEST_BOARD="fL7fv67z"
-RUBY=$(which ruby)
+RUBY="/usr/bin/ruby.ruby2.2"
```

Comment:
This will not work on half of the machines and once we get newer workers it would need to be updated - e.g. Factory has ruby2.4 but no ruby2.2